mps-sync-plugin/src/main/kotlin/org/modelix/mps/sync/plugin/gui/ModelSyncGuiFactory.kt

```diff
@@ -214,6 +214,14 @@ class ModelSyncGuiFactory : ToolWindowFactory, Disposable {
             branchPanel.add(JLabel("Remote Branch: "))
             branchPanel.add(branchCB)
             inputBox.add(branchPanel)
+            val connectBranchButton = JButton("Connect to Branch without downloading Modules")
```

Comment:
`Model-server --> MPS`: The "Bind Selected" starts with a module from the server. Without it there is no ReplicatedModel locally. 


`MPS --> Model-server`: The "Connect to Branch without downloading Modules" connects to a Branch to get a RM, but does not sync its content locally. It's a bit dangerous at the moment, because it could introduce inconsistencies (the user does not see everything in MPS from the branch, but modelix has all things locally). So that parts have to be cleaned up.